Fix dev branch

Tschakki opened the pull request:
Fix conflicts

RachBLondon commented on examples.md:
```diff
@@ -7,7 +7,7 @@
 Welcome to the step-by-step guide of creating the Hello World application with Lisk Alpha SDK.
 A simple App, showcasing a minimal setup of a blockchain application with 1 [custom transaction](custom-transactions.md) type: the "Hello" transaction.
```
```suggestion
A simple App, showcasing a minimal setup of a blockchain application with 1 [custom transaction](custom-transactions.md) type: the "HelloWorld" transaction.
```

RachBLondon commented on examples.md:
```diff
@@ -7,7 +7,7 @@
 A simple App, showcasing a minimal setup of a blockchain application with 1 [custom transaction](custom-transactions.md) type: the "Hello" transaction.
 
-The purpose of Hello World application is to explain how to use and how to implement custom transaction with the Lisk SDK. 
+The purpose of Hello World application is to explain how to use and how to implement custom transactions with the Lisk SDK. 
```
```suggestion
The purpose of Hello World application is to show how to implement custom transactions with the Lisk SDK. 
```

RachBLondon commented on examples.md:
```diff
@@ -7,7 +7,7 @@
 
-The purpose of Hello World application is to explain how to use and how to implement custom transaction with the Lisk SDK. 
+The purpose of Hello World application is to explain how to use and how to implement custom transactions with the Lisk SDK. 
 The implementation is saving the string value of the "hello" transaction's asset property to the asset property of the sender's account.
```
```suggestion
The implementation is saving the string value of the "helloWorld" transaction's asset property to the asset property of the sender's account.
```

RachBLondon commented on examples.md:
```diff
@@ -396,14 +397,238 @@ As next step, you can design a nice frontend application like [Lisk Explorer](ht
+
+#### TYPE
+
+Set the Cashback transaction TYPE to `11`.
```
Again should reserve more than 10 for Lisk?

RachBLondon commented on examples.md:
```diff
@@ -396,14 +397,238 @@ As next step, you can design a nice frontend application like [Lisk Explorer](ht
+
+So e.g. if Alice sends 100 token to Bob as a Cashback transaction, Bob would receive the 100 token and Alice would receive additional 10 tokens as a cashback.
+
+> If you compare the methods below with the methods we implemented for the `HelloTransaction`, you will notice, that we implement less methods for the `CashbackTransaction`.
```
```suggestion
> If you compare the methods below with the methods we implemented for the `HelloWorldTransaction`, you will notice, that we implement less methods for the `CashbackTransaction`.
```
If we want to move to the HelloWorld name

RachBLondon commented on examples.md:
```diff
@@ -396,14 +397,238 @@ As next step, you can design a nice frontend application like [Lisk Explorer](ht
+
+#### applyAsset
+
+It sends the transaction amount to the recipient and substracts 1/10 of the transaction amount from the senders account balance
```
```suggestion
The CashbackTransaction adds an inflationary 10% to senders account.
```

RachBLondon commented on examples.md:
```diff
@@ -396,14 +397,238 @@ As next step, you can design a nice frontend application like [Lisk Explorer](ht
+
+const app = new Application(genesisBlockDevnet, configDevnet); // create the application instance
+
+app.registerTransaction(11, CashbackTransaction); // register the 'CashbackTransaction' 
```
```suggestion
app.registerTransaction(CashbackTransaction); // register the 'CashbackTransaction' 
```
According @nazarhussain we can remove the number, is this correct @MaciejBaj ?

RachBLondon commented on examples.md:
````diff
@@ -396,14 +397,238 @@ As next step, you can design a nice frontend application like [Lisk Explorer](ht
+const app = new Application(genesisBlockDevnet, configDevnet); // create the application instance
+```
+
+In the `line 2`, we require the needed dependencies from the `lisk-sdk` package.
````
```suggestion
On `line 2`, we require the needed dependencies from the `lisk-sdk` package.
```

RachBLondon commented on examples.md:
```diff
@@ -396,14 +397,238 @@ As next step, you can design a nice frontend application like [Lisk Explorer](ht
+
+It sends the transaction amount to the recipient and substracts 1/10 of the transaction amount from the senders account balance
+
+Invoked as part of the `apply` step of the BaseTransaction and block processing.  
```
```suggestion
```
I think this sentence is a little confusing

MaciejBaj replied:
There is an apply step on base transaction as well, applyAsset is being called as part of it.

TalhaMaliktz committed:
Release 13-11-2024 (#244)

* [LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* [LISKDOCS-23] Update favicon (#237)

Update favicon

* [LISKDOCS-24]Update typography for mobile-view (#239)

Update typography for mobile-view

* Resolve conflicts in development (#241)

* Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove line

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

* Release 12-11-2024 (#236)

[LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* Release 2: 12-11-2024 (#238)

* [LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* [LISKDOCS-23] Update favicon (#237)

Update favicon

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

* Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove line

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

* Remove comment and coming soon

* Resolve conflicts from main (#242)

* Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove line

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

* Release 12-11-2024 (#236)

[LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* Release 2: 12-11-2024 (#238)

* [LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* [LISKDOCS-23] Update favicon (#237)

Update favicon

* Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove line

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

* Remove comment and coming soon

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

Tschakki committed:
Merge branch 'main' into development

TalhaMaliktz reviewed (changes requested):
The file has the `@media (max-width: 576px) {` section twice! this change should be removed line 45-86

Tschakki committed:
Remove duplicate
